### PR TITLE
[onert/odc] Introduce Quantizer

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit introduces Quantizer as onert module.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: https://github.com/Samsung/ONE/issues/11391
Draft: https://github.com/Samsung/ONE/pull/11298

### CODE REVIEW

**Comment by hseok-oh** [comment]
@nnfw-bot test onert-x64-release

**Comment by glistening** [comment]
@nnfw-bot test onert-x64-release

**Review by glistening — APPROVED** [review]
LGTM

File: runtime/onert/odc/Quantizer.test.cc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{
+  Quantizer quantizer;
+  ASSERT_THROW(quantizer.quantize(nullptr, "out", false), std::logic_error);
+}
+
+// Test model output path is not set
+TEST(odc_Quantizer, neg_model_output_path)
+{
+  Quantizer quantizer;
+  ASSERT_EQ(quantizer.quantize("in", nullptr, false), 1);
```
Comment (ragmani) [inline runtime/onert/odc/Quantizer.test.cc:34]: (Optional)
How about using `ASSERT_NE`?
```suggestion
  ASSERT_NE(quantizer.quantize("in", nullptr, false), 0);
```

File: runtime/onert/odc/Quantizer.test.cc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+// Test invalid model input path
+TEST(odc_Quantizer, neg_invalid_model_input_path)
+{
+  Quantizer quantizer;
+  ASSERT_EQ(quantizer.quantize("invalid_model_input_path.circle", "out", false), 1);
```
Comment (ragmani) [inline runtime/onert/odc/Quantizer.test.cc:41]: ditto.

**Review by ragmani — APPROVED** [review]
LGTM

**Comment by hseok-oh** [comment]
I'll update unittests after merging.

**Pull request merged**